test/unittests/evm_state_test.cpp

```diff
@@ -95,42 +95,72 @@ TEST_F(evm_state, sstore_cost)
         storage[v1] = v1;
         execute(sstore(1, push(1)));
         EXPECT_EQ(result.status_code, EVMC_SUCCESS);
-        EXPECT_EQ(gas_used, rev == EVMC_CONSTANTINOPLE ? 206 : 5006);
+        if (rev == EVMC_CONSTANTINOPLE)
+            EXPECT_EQ(gas_used, 206);
+        else if (rev == EVMC_ISTANBUL)
+            EXPECT_EQ(gas_used, 806);
+        else
+            EXPECT_EQ(gas_used, 5006);
         execute(205, sstore(1, push(1)));
         EXPECT_EQ(result.status_code, EVMC_OUT_OF_GAS);
 
         // Added & unchanged:
         storage.clear();
         execute(sstore(1, push(1)) + sstore(1, push(1)));
         EXPECT_EQ(result.status_code, EVMC_SUCCESS);
-        EXPECT_EQ(gas_used, rev == EVMC_CONSTANTINOPLE ? 20212 : 25012);
+        if (rev == EVMC_CONSTANTINOPLE)
+            EXPECT_EQ(gas_used, 20212);
+        else if (rev == EVMC_ISTANBUL)
+            EXPECT_EQ(gas_used, 20812);
+        else
+            EXPECT_EQ(gas_used, 25012);
 
         // Modified again:
         storage.clear();
         storage[v1] = {v1, true};
         execute(sstore(1, push(2)));
         EXPECT_EQ(result.status_code, EVMC_SUCCESS);
-        EXPECT_EQ(gas_used, rev == EVMC_CONSTANTINOPLE ? 206 : 5006);
```

Comment:
Shouldn't all these have `>= CONSTANTINOPLE`?

E.g. `(rev >= istanbul) elseif (rev == constantinople) else ..` (because it was disabled in petersburg)

Reply:
right, changed now to cover post-Istanbul revisions